tests/remote/resources.tf

```diff
@@ -0,0 +1,28 @@
+variable "location" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+resource "azurerm_resource_group" "tftest" {
+  name     = var.resource_group_name
```

Comment:
this needs to be a random resource_group_name, to run multiple tests and decouple them.

Reply:
See [here](https://github.com/cloudeteer/terraform-azurerm-vm/blob/559260ffe05607995dd67fc02c6e1d4acd019222/tests/remote/main.tf#L37-L49) for a working example.